src/api/xccl.h

```diff
@@ -88,21 +88,27 @@ typedef enum {
  * team context (i.e., a mapping to the world rank)
  * The number of endpoints is provided as part of @ref xccl_team_config_t.
  */
+struct xccl_ep_range_strided {
```

Comment:
@Sergei-Lebedev do u think we need these 3 structures defined separately?

Reply:
Nope, but the reason I had to move them outside is because cpp compiler (nvcc) doesn't allow structure definition inside anonymous union